packages/onboarding/src/setup-tailored-site-after-creation.ts

```diff
@@ -93,13 +93,37 @@ export function setupSiteAfterCreation( { siteId, flowName }: SetupOnboardingSit
 		return Promise.resolve();
 	};
 
+	const removeInitialPost = async ( siteId: number, flowName: string ) => {
+		if ( flowName === NEWSLETTER_FLOW ) {
+			// We are deleting the initial "Hello World!" post for newsletter sites
+			// The id for the first post on a new site will be 3
```

Comment:
Will it always be 3? I wonder if this could change.

Reply:
I'm actually curious about this too. 

@alshakero - Are you able to provide some context or info regarding a new site's first post id = 3?